test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1

```diff
@@ -35,4 +35,17 @@ Describe 'ConvertTo-Json' -tags "CI" {
         $ps.InvocationStateInfo.State | Should -BeExactly "Stopped"
         $ps.Dispose()
     }
+
+    It "The result string is packed in an array symbols when AsArray parameter is used." {
+        $output = ConvertTo-Json -InputObject 1 -AsArray
+        $output | Should -BeLike "``[*1*]"
+
+        $output = ConvertTo-Json -InputObject 1,2 -AsArray
```

Comment:
This already works:

```powershell
ConvertTo-Json -InputObject 1,2 
```

What needs to be tested is pipeline input:

```powershell
$output = 1,2 | ConvertTo-Json
```

Please adjust tests